### ISSUE TITLE
Webapp > Help > Text size understanding : comments on pt => px conversion

### ISSUE THREAD

**Issue by ArnaudDelafosse** [Asqatasun/Contrast-Finder#136]
## Observed behaviour
In the "[Help](https://app.contrast-finder.org/#user-help-contrast-finder)" > "Minimum ratio" > " Level AA" and "Text size understanding" sections, it is implied that 14pt is equivalent to 14px and that 18pt is equivalent to 18px.

## Expected behaviour
I believe this is incorrect if you follow the W3C's own "[Contrast (Minimum) Understanding SC 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)" in the "Note 1" section where it says "The ratio between sizes in points and CSS pixels is 1pt = 1.333px, therefore 14pt and 18pt are equivalent to approximately 18.5px and 24px."

What I understand is that for a given standard font-size where `body { font-size: 100%; } p { font-size: 1rem; }` sets your paragraphs a font-size of (equivalent to) 16px, contrast ratios for level AA should be at least:
- 4,5 for body copy,
- 3 for text in bold with a font-size of at least (equivalent to) 18.5px or 120% of body copy,
- 3 for large text with a font-size of at least (equivalent to) 24px or 150% of body copy. 

/!\ Reminder: do **not** use px to set your font sizes, these are pixel equivalences. Always use relative units like **rem**, **em** or **%**.

## Steps to reproduce the behaviour
See this example on CodePen: https://codepen.io/ArnzWeb/full/WMdmGB/ (it's in French and I've voluntarily rounded 18.5 to 19px).

**Comment by mfaure** [comment]
Thanks @ArnaudDelafosse for your clever feedback! This note wasn't present at the time of the doc writing. I just grep'd some git logs and found it was introduced on [May 23, 2016](https://github.com/w3c/wcag/commit/fbdeb866068188e14b1ba0e81f74416df413e9b7#diff-aca735726929048d79c6f8e6f0a08882).

Would you feel comfortable with fixing the doc ?

**Comment by ArnaudDelafosse** [comment]
You're welcome, glad to bring a bit of help to this great tool.
Yes I'm fine with amending the doc, I'll try as soon as I can.

**Comment by dzc34** [comment]
files impacted:
- [ ] [documentation/en/Howto-use.md](https://github.com/Asqatasun/Contrast-Finder/blob/develop/documentation/en/Howto-use.md)  ---> PR #143
- webapp (HTML in i18n file)
  - [ ] [English i18n file](https://github.com/Asqatasun/Contrast-Finder/blob/develop/webapp/src/main/resources/i18n/contrast-finder_utf8_en.properties#L99)
  - [ ] [French i18n file](https://github.com/Asqatasun/Contrast-Finder/blob/develop/webapp/src/main/resources/i18n/contrast-finder_utf8_fr.properties#L98)
  - [ ] [Korean i18n file](https://github.com/Asqatasun/Contrast-Finder/blob/develop/webapp/src/main/resources/i18n/contrast-finder_utf8_ko.properties#L99)

**Comment by ArnaudDelafosse** [comment]
Once the suggestions I made for the Howto page are validated or not, I'll mirror that on the 2 remaining files.

**Comment by dzc34** [comment]
Thanks @ArnaudDelafosse for your contributing! 
I splited your merge request #140 in two part for more easy review:
- [X] #142  - DOC - Howto-use.md improvements (syntax corrections, match titles to application labels) 
- [ ] #143  - DOC - Howto-use.md / Update for the pt => px conversion section

#142 is now merged in develop branch.

**Comment by dzc34** [comment]
other resources:
- [w3c/wcag - "Large text" - using px rather than pt as unit](https://github.com/w3c/wcag/issues/181)
- [DISIC/rgaa_referentiel  - Meilleure indication de la taille de caractères pour le calcul des contrastes](https://github.com/DISIC/rgaa_referentiel/issues/52)
